apis/eseries/eseries.go

```diff
@@ -111,7 +118,9 @@ func (d Driver) SendMsg(data []byte, sendType string, msgType string) (*http.Res
 	req.SetBasicAuth(d.config.Username, d.config.Password)
 
 	tr := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: d.config.WebProxyInsecure,
```

Comment:
I think you are right in having two configuration settings (one controlling http vs. https and the other for skipping TLS verification). 

I like your variable names. The only thing I would alter here is defaulting webProxySkipTLSVerify to be true rather than false. I know defaulting to true would make man-in-the-middle attacks easier, but I am worried that defaulting to false would cause panics for users that don't have certificates signed by a legitimate certificate authority. For those users that want the extra security here then they cause alter the default value in the configuration.